packages/peregrine/jest.config.js

```diff
@@ -1,11 +1,12 @@
 module.exports = {
+    browser: true,
+    coveragePathIgnorePatterns: ['scripts/*', 'node_modules', 'src/index.js'],
     displayName: 'Peregrine',
     setupFiles: [
         '<rootDir>/scripts/shim.js',
         '<rootDir>/scripts/fetch-mock.js'
     ],
-    browser: true,
-    testURL: 'https://localhost/',
-    coveragePathIgnorePatterns: ['scripts/*', 'node_modules', 'src/index.js'],
-    testPathIgnorePatterns: ['dist', 'node_modules']
+    setupTestFrameworkScriptFile: '<rootDir>/scripts/jest-setup.js',
```

Comment:
The addition of this property is really the only change in this file. The rest are just alphabetical order changes.